.github/actions/create-deployment-package/action.yml

```diff
@@ -3,9 +3,6 @@ description: "Create a deployment package for Headless and Atomic"
 runs:
   using: composite
   steps:
-    - uses: actions/setup-python@v2
```

Comment:
Presumably not needed anymore, since the sample uses the EC2 runner and didn't install Python.